src/DynamoUtilities/Md2Html.cs

```diff
@@ -104,7 +102,7 @@ internal string SanitizeHtml(string content)
 
             var output = GetData(processCommunicationTimeoutms);
 
-           return output.Result;
```

Comment:
would an alternative be to always access this data from another thread, for example - like we do with the feature flags manager startup - run the GetData call inside on a task in the thread pool?

Reply:
It might work. But having a separate thread started everywhere GetData.Result is called might be harder to maintain
I find it easier to reason about when the entire concurrency logic is boxed up inside the GetData method so callers do not need to worry about how to call it

Reply:
I actually used the approach you suggested for another case (in a test file)
https://github.com/DynamoDS/Dynamo/pull/14649/files#diff-be236d5cf3315a8f37d3d67a4adb628825430289b534dbd620952c47f2d286b0R1136
For production code (like GetData) I still think it would make it tricky to figure out how to use it safely.